examples/onnxruntime/training/image-classification/README.md

```diff
@@ -0,0 +1,54 @@
+<!---
+Copyright 2022 The HuggingFace Team. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Language Modeling
+
+## Image Classification Training
+
+By running the scripts [`run_image_classification.py`](https://github.com/huggingface/optimum/blob/main/examples/onnxruntime/training/image-classification/run_image_classification.py) we will be able to leverage the [`ONNX Runtime`](https://github.com/microsoft/onnxruntime) accelerator to train the language models from the
+[HuggingFace hub](https://huggingface.co/models).
+
+
+__The following example applies the acceleration features powered by ONNX Runtime.__
+
+
+### ONNX Runtime Training
+
+The following example trains ViT on beans dataset with mixed precision (fp16).
+
+```bash
+python run_image_classification.py \
+    --model_name_or_path google/vit-base-patch16-224-in21k \
+    --dataset_name beans \
+    --output_dir ./beans_outputs/ \
+    --remove_unused_columns False \
+    --do_train \
+    --do_eval \
+    --learning_rate 2e-5 \
+    --num_train_epochs 10 \
+    --per_device_train_batch_size 32 \
+    --per_device_eval_batch_size 32 \
+    --logging_strategy steps \
+    --logging_steps 10 \
+    --evaluation_strategy epoch \
+    --seed 1337
+```
+
+
+__Note__
+> *To enable ONNX Runtime training, your devices need to be equipped with GPU. Install the dependencies either with our prepared*
+*[Dockerfiles](https://github.com/huggingface/optimum/blob/main/examples/onnxruntime/training/docker/) or follow the instructions*
+*in [`torch_ort`](https://github.com/pytorch/ort/blob/main/torch_ort/docker/README.md).*
+
+> *The inference will use PyTorch by default, if you want to use ONNX Runtime backend instead, add the flag `--inference_with_ort`.*
```

Comment:
The support of ONNX export for VIT is not enabled yet. Can you remove `--inference_with_ort` note from README?

Reply:
@JingyaHuang I tested ViT and I got ~5% gain from ORT with this command: python -m torch.distributed.launch --nproc_per_node=8 ~/optimum/examples/onnxruntime/training/image-classification/run_image_classification.py --model_name_or_path google/vit-base-patch16-224-in21k --dataset_name beans --overwrite_output_dir --output_dir ./outputs/ --remove_unused_columns False --do_train --do_eval --learning_rate 2e-5 --num_train_epochs 50 --per_device_train_batch_size 8  --per_device_eval_batch_size 8 --logging_strategy steps --logging_steps 10 --evaluation_strategy epoch --seed 1337 --fp16 True  --report_to none --optim adamw_ort_fused

Regardless though, I removed the --inference_with_ort flag since it isn't required for runs with optimum

Reply:
I suppose the gain is in training. Yeah `--inference_with_ort` is not implemented in the script so better remove it. Thanks.